README.md

```diff
@@ -113,6 +112,26 @@ Then run the following:
   - Opera
     - `yarn run dev:opera`
 
+### :white_check_mark: Tests
+
+#### E2E tests via playwright
+
+```bash
+yarn run dev:chrome
+yarn playwright test
+```
+
+:warning:
+
+- You need to first import the extension into the browser(s)
+- You might need to adjust the `extensionID` in playwright spec(s)
+
+#### Unit tests tests via Jest
+
+```bash
+yarn test
```

Comment:
We could run all tests here and split it internally to have i.e.
- `yarn test:jest`
- `yarn test:playwright`

Reply:
+1